### PR TITLE
Use Body.from(Object) instead of Body.from(Map<String,Object>)

### PR DESCRIPTION
**Pull request by smhc**
To resolve https://github.com/openapi4j/openapi4j/issues/86

### CODE REVIEW

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java [hunk 1]
```diff
@@ -8,41 +8,42 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.List;
```
Comment (llfbandit) [inline openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java]: should be removed

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java [hunk 2]
```diff
@@ -61,13 +62,13 @@ public static Body from(JsonNode body) {
...
   /**
-   * Constructs a body from the given map tree model.
+   * Constructs a body from the given Java Object.
    * This is the preferred way to build a body wrapper.
    *
    * @param body the given map tree model (JSON, XML, ...)
    * @return The constructed body.
    */
-  public static Body from(Map<String, Object> body) {
+  public static Body from(Object body) {
```
Comment (llfbandit) [inline openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java]: Java doc should be updated to reflect the changes (look for 'map')

**Comment by smhc** [comment]
I tidied it up - although I'm not sure what is best for the JavaDoc. It seems that TreeUtil.json.convertValue method can take a wide variety of inputs. I am also not clear on XML handling and why that is mentioned in the doc.

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java (hunk 2 above)
Comment (smhc) [reply openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java]: I tidied it up - although I'm not sure what is best for the JavaDoc. It seems that TreeUtil.json.convertValue method can take a wide variety of inputs. I am also not clear on XML handling and why that is mentioned in the doc.

File: openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java (hunk 1 above)
Comment (smhc) [reply openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/model/impl/Body.java]: removed.

**Pull request merged**